### PR TITLE
:tada: Listen for all events from the p4nth3rb0t-mainframe

### PR DESCRIPTION
**Pull request by whitep4nth3r**
Also removed UserManager as the mainframe takes care of that now.

### CODE REVIEW

**Comment by whitep4nth3r** [comment]
@all-contributors please add @LuceCarter for ideas

**Comment by whitep4nth3r** [comment]
@all-contributors please add @gigili for ideas

File: package.json [hunk 1]
```diff
@@ -6,8 +6,7 @@
     "p5": "^1.1.9",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-scripts": "3.4.3",
-    "tmi.js": "^1.5.0"
```
Comment (isabellabrookes) [inline package.json:10]: @whitep4nth3r I don't see the corresponding yarn.lock being pushed.

**Comment by whitep4nth3r** [comment]
Curious. Running a yarn upgrade now to see what happens...

**Pull request merged**